core/modules/field/modules/number/number.module

```diff
@@ -203,6 +203,8 @@ function number_field_formatter_settings_form($field, $instance, $view_mode, $fo
   $settings = $display['settings'];
   $element = array();
 
+  $element = array();
```

Comment:
This is from https://www.drupal.org/node/1732906 / http://cgit.drupalcode.org/drupal/commit/?id=11dac6e but I don't think we need it (same line just 2 lines above). Must have fixed it at some earlier point.